support-workers/src/main/scala/com/gu/salesforce/SalesforceService.scala

```diff
@@ -177,9 +177,7 @@ class AuthService(config: SalesforceConfig)(implicit ec: ExecutionContext)
         "grant_type" -> Seq("password")
       ))
 
-    Retry(3) {
-      postAuthRequest
-    }
+    postAuthRequest
```

Comment:
I think the whole lambda would retry anyway if there was an auth failure?  If not, it would still be more sensible to retry at that level.  I noticed this because when the test was deadlocking, with a 30 second timeout, it was actually taking >90 seconds.  I then noticed the retrying in the test logs.